PSKoans/Koans/Constructs and Patterns/AboutSplatting.Koans.ps1

```diff
@@ -65,6 +67,7 @@ Describe 'Splatting' {
             $Value = __
             if ($Value -eq 7) {
                 $Parameters.Add('File', $true)
+                $Parameters.Add('Recurse', $true)
```

Comment:
Why add the -Recurse here?

Reply:
Because there are no files at the root of $PSKoansFolder.